openmetadata-ui/src/main/resources/ui/src/components/AddGlossaryTermPage/AddGlossaryTermPage.component.tsx

```diff
@@ -52,35 +53,57 @@ const AddGlossaryTermPage = () => {
     goToGlossary();
   };
 
+  const handleShowErrorToast = (errMessage: string) => {
```

Comment:
Can we move this to a utility file to reuse across different components to handle errors?

Reply:
It uses `showToast`, which is from `useToastContext()` custom hook, and hooks cannot be used outside the scope of a `FunctionComponent`